scripts/setup/rewrite_yaml_files.sh

```diff
@@ -0,0 +1,320 @@
+
+
+# serving-crds.yaml
+
+cat serving-crds.yaml | sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' > serving-crds-yq.yaml
+
+# serving-default-domain.yaml
+
+cat serving-default-domain.yaml | 
+    yq ' 
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "default-domain"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/default-domain-dd6f55160290a90018ba318b624be12e@sha256:0206e792ee18591903e8b8aaaba92d04548c9b63d9742a98195037697bdbc42e"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-default-domain-yq.yaml
+
+# serving-hpa.yaml
+
+cat serving-hpa.yaml |  
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "autoscaler-hpa"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/autoscaler-hpa-85c0b68178743d74ff7f663a72802ceb@sha256:1797f6c16f0c65b0255651f82e85399ba210cc353044177ad2af159fc20e2122"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-hpa-yq.yaml
+
+# serving-post-install-jobs.yaml
+
+cat serving-post-install-jobs.yaml |
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "storage-version-migration-serving"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/migrate-242d0a35bf580c5b411a545d79618fbf@sha256:2d2159119a4c3bd8470a14a970835aa1f56a64250a9ff8bc59ea534175f8548b"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-post-install-jobs-yq.yaml
+
+# serving-storage-version-migration.yaml
+
+cat serving-storage-version-migration.yaml |
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "storage-version-migration-serving"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/migrate-242d0a35bf580c5b411a545d79618fbf@sha256:2d2159119a4c3bd8470a14a970835aa1f56a64250a9ff8bc59ea534175f8548b"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-storage-version-migration-yq.yaml
+
+
+# canal.yaml
+
+MASTER_NODE_IP=$(ip route | awk '{print $(NF)}' | awk '/^10\..*/')
+IFACE=$(netstat -ie | grep -B1 $MASTER_NODE_IP | head -n1 | awk '{print $1}' | cut -d ':' -f 1)
+
+cat ../calico/canal.yaml | \
+    yq '
+    (
+        select
+        (
+            .spec.template.spec.containers[].name == "calico-node"
+        ) | .spec.template.spec.containers[0].env 
+    ) |= . + [ {"name": "CALICO_IPV4POOL_CIDR", "value": "10.168.0.0/16"} ]' | 
+    yq '
+    (
+        select
+        (           
+                .metadata.name == "calico-config"
+            and .metadata.namespace == "kube-system"
+        ) | .data.canal_iface
+    ) = "_IFACE_"' |
+    # yq '
```

Comment:
why is it commented? remove?

Reply:
This parameter is no longer used. This parameter is currently not used in vhive and invitro.